### ISSUE TITLE
how to load a trained model and continue to train with a different loss correctly

### ISSUE THREAD

**Issue by haixpham** [Microsoft/CNTK#2616]
I have a fairly complex network, and it needs to be trained for a long time. However, I want to change the loss trade-off weights and continue to train from the already pretrained weights. The model is reloaded as in the following pseudocode:

```python
def create_model_from_pretrained(input_1, input_2, model_file):
     trained_model = C.load_model(model_file)
     input1_node = trained_model.find_by_name('input_1')
     input2_node = trained_model.find_by_name('input_2')
     cloned_model = trained_model.clone(C.CloneMethod.clone, {input1_node: input_1, input2_node: input_2})
     return cloned_model
```

After this, I just use the returned function in loss function and learner/trainer respectively.

However, I get totally unexpected result, compared to the output of the previously trained network.

Did I make mistake here?

**Comment by aWeinzierl** [comment]
Have you tried without cloning and just using C.Load_model(model_file)?

**Comment by haixpham** [comment]
It seems the mistake is in my training criteria, and not that part of loading pretrained weights. It took several trials to confirm.
